src/main/resources/assets/matteroverdrive/shaders/core/android_station_shader.fsh

```diff
@@ -25,9 +25,9 @@ void main() {
     }
     color *= vertexColor * ColorModulator;
 
-    float timer = (sin(GameTime * 2400 * 0.5) + 1) / 4 + 0.7;
+    float timer = (sin(GameTime * 2400 * 0.5) + 1) / 4 + 0.7;t
```

Comment:
Added unnecessary "t"

Reply:
Thats weird thats not in my local one :thinking:  must have merged odd